go.sum

```diff
@@ -116,8 +116,12 @@ github.com/codeready-toolchain/api v0.0.0-20200226022556-bb2d9dd17828 h1:lWjwDBd
 github.com/codeready-toolchain/api v0.0.0-20200226022556-bb2d9dd17828/go.mod h1:w07CIyd2cCp1i38hdPrG4aZPbwISzMkxOmtCWE4EtdY=
 github.com/codeready-toolchain/api v0.0.0-20200226154004-addd941b675c h1://VqLgLGvtUiWcJn3qju6PjIoc01O+QrVD19oK/s8Zk=
 github.com/codeready-toolchain/api v0.0.0-20200226154004-addd941b675c/go.mod h1:w07CIyd2cCp1i38hdPrG4aZPbwISzMkxOmtCWE4EtdY=
+github.com/codeready-toolchain/api v0.0.0-20200305152414-a958daa81263 h1:LyzNCGqUwk6j9q48vFdLriHPEyGsZcQcvOtq/ZbHA6w=
```

Comment:
you could use `go mod tidy` to remove unused dependencies.

Reply:
done in 32664fb